astrodynamics-fall-2018/astrodynamicsFall2018PDD.tex

```diff
@@ -0,0 +1,372 @@
+
+  \begin{table}[hb!]
+    % this table is too wide for the two-column format, so we let it expand across both columns
+    % we haven't told LaTeX where to put this so it'll find the best place.
+        \caption{Estimated Budget}
+        \centering
+        \begin{tabularx}{\linewidth}{lXl}
+            % READ THIS!! https://www.inf.ethz.ch/personal/markusp/teaching/guides/guide-tables.pdf
+            \toprule % line on top external edge of table
+            % Separate cells in a row with &, move to the next row with \\
+            Project & Cost \\
+            \midrule % line separating two internal rows
+            Roll Off Motor Repair & 500 \\
+            Motor Latch Arm & 100 \\
+            12" Telescope Modifications & 100 \\
+            % LaTeX doesn't really like multi-line cell contents. Try to keep the text in each cell concise!
+            \bottomrule
+        \end{tabularx}
+    \label{tab:costEstimate}
+    \end{table}
+
+\subsection{Faculty Support}
+  % Identify faculty that will be involved (or would need to be involved) to meet objectives.
+  % Note that if a professor is the Principal Investigator (P.I.) for a project, there still needs to be a student as the SPEX Project Champion.
+  Direct support from Dr. Richmond and Dr. Connelly is expected.
+  Access to the facility is highly dependent on one of the above being present.
+  Both faculty are very familiar with the equipment, telescopes, and facility.
+  They also have experience with what is typical for systems at other observatories, which may be beneficial when working on these projects.
+  Dr. Richmond has been modifying equipment at the observatory for decades.
+  Dr. Connelly understands the structure and capabilities of RIT SPEX through being an advisor.
+  Both are great resources for guidance during the completion of the projects.
+  As all professors, they are juggling multiple responsibilities at one time, which can complicate scheduling time for this project.
+  Even so, they understand the needs of the observatory and can also be seen as the primary customers for these projects, so it is important to keep them informed through the engineering process
+
+\subsection{Long-Term Vision}
+\label{sec:vision}
+The long term goal of these projects is to work towards a fully automated dome system.
+This would enable remote observing for the Observatory and enable increased access to more students.
+Students will get familiar with the technology behind observing and enhance their basic engineering skills.
+As projects get completed, a strong bond between the RIT Observatory and RIT SPEX can be established.
+
+\section*{Acknowledgements}
+The author would like to thank the advisors of SPEX, Evan Putnam and Marilyn Wolbert for continuing the Astrodynamics team for the 2017-2018 school year, the faculty at theobservatory for reaching out about these projects, and the SPEX admins for their extra effort put in to make this project process possible.
```

Comment:
the proper dash here is an "en dash" rather than a hyphen, but that's getting picky. since i already wrote this comment, change `2017-2018` to `2017--2018`